client/app/i18n/locales/de/Dashboards.json

```diff
@@ -0,0 +1,83 @@
+  "Download as CSV File": "Als CSV-Datei herunterladen",
+  "Download as TSV File": "Als TSV-Datei herunterladen",
+  "Download as Excel File": "Als Excel-Datei herunterladen",
+  "View Query": "Abfrage anzeigen",
+  "Edit_Parameters": "Parameter bearbeiten",
+  "Refreshing...": "Wird aktualisiert...",
+  "Refreshing, please wait. ": "Wird aktualisiert, bitte warten.",
+  "Press to refresh. ": "Drücken Sie zum Aktualisieren.",
+  "Refreshing, please wait.": "Wird aktualisiert, bitte warten.",
+  "Press to refresh.": "Drücken Sie zum Aktualisieren.",
+  "Error running query": "Fehler bei der Abfrageausführung",
+  "Remove_from_Dashboard": "Vom Dashboard entfernen",
+  "More_options": "Mehr Optionen",
+  "Remove From Dashboard": "",
+  "Close": "",
+  "Delete Widget": "Widget löschen",
+  "Are you sure you want to remove this widget from the dashboard?": "Sind Sie sicher, dass Sie dieses Widget vom Dashboard entfernen möchten?",
+  "Widget cannot be updated": "Widget kann nicht aktualisiert werden",
+  "Widget could not be saved": "Widget konnte nicht gespeichert werden",
+  "Quit editing?": "Bearbeitung beenden?",
+  "Changes you made so far will not be saved. Are you sure?": "Die bisherigen Änderungen werden nicht gespeichert. Sind Sie sicher?",
+  "Yes, quit": "Ja, beenden",
+  "Add Textbox": "Textbox hinzufügen",
+  "Edit Textbox": "Textbox bearbeiten",
+  "Textbox widget content": "Textbox-Widget-Inhalt",
+  "This is where you write some text": "Hier können Sie Text eingeben",
+  "Supports basic": "Unterstützt grundlegendes",
+  "Markdown guide opens in new window": "Markdown-Anleitung öffnet sich in einem neuen Fenster",
+  "Markdown": "Markdown",
+  "Preview": "Vorschau",
+  "Auto Refreshing every {{duration}}": "Automatisches Aktualisieren alle {{duration}}",
+  "Refresh": "Aktualisieren",
+  "Disable auto refresh": "Automatisches Aktualisieren deaktivieren",
+  "Split button!": "Split-Button!",
+  "Archive Dashboard": "Dashboard archivieren",
+  "Are you sure you want to archive the {{dashboardName}} dashboard?": "Sind Sie sicher, dass Sie das Dashboard {{dashboardName}} archivieren möchten?",
+  "Enable/Disable Fullscreen display": "Vollbildanzeige aktivieren/deaktivieren",
+  "Toggle fullscreen display": "Vollbildanzeige umschalten",
+  "Dashboard Sharing Options": "Dashboard-Freigabeoptionen",
+  "Share": "Teilen",
+  "Saved": "Gespeichert",
+  "Saving": "Speichern",
+  "Saving Failed": "Speichern fehlgeschlagen",
+  "Retry": "Erneut versuchen",
+  "Done Editing": "Bearbeitung abgeschlossen",
+  "Sorry, we couldn't find anything.": "Entschuldigung, wir konnten nichts finden.",
+  "Mark dashboards as Favorite to list them here.": "Markieren Sie Dashboards als Favoriten, um sie hier aufzulisten.",
+  "Create your first dashboard!": "Erstellen Sie Ihr erstes Dashboard!",
+  "Need help?": "Brauchen Sie Hilfe?",
+  "See the big picture": "Den großen Überblick sehen",
```

Comment:
Apparently, it is this screen when there is no dashboard created yet, so yeah i think we can leave it as it is "Den großen Überblick sehen", because it is not even clickable , thanks a lot Guelden for checking it.

<img width="1427" alt="Screenshot 2024-11-28 at 16 40 23" src="https://github.com/user-attachments/assets/42b74886-efca-4ef7-9eb1-0534999d795e">